host-configs/lc-builds/blueos/xl_2020_X.cmake

```diff
@@ -7,10 +7,10 @@
 
 set(RAJA_COMPILER "RAJA_COMPILER_XLC" CACHE STRING "")
 
-set(CMAKE_CXX_FLAGS_RELEASE "-O3 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qnoeh -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
-set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qnoeh -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
-set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g -qsmp=omp:noopt " CACHE STRING "")
-set(CMAKE_EXE_LINKER_FLAGS "-Wl,-z,muldefs" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELEASE "-O3 -std=c++14 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g -std=c++14 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g -std=c++14 -qsmp=omp:noopt " CACHE STRING "")
```

Comment:
BLT_CXX_STD?

Reply:
In a reply to your related comment:
"For some reason, doing set(BLT_CXX_STD c++14 CACHE STRING "") only adds -std=c++1y to the compilation line. Let's leave -std=c++14 in for now."

Reply:
That's because that's what BLT thinks xl understands, if it should be c++14 for newer versions then that change should be made in blt. I'm not sure what copmilers do when they get more than one std flag, it seems like we should avoid that.

Reply:
@davidbeckingsale Would you mind changing the BLT_CXX_STD for XL to `-std=c++14`? It should be a one-line change here: https://github.com/LLNL/blt/blob/develop/cmake/SetupCompilerOptions.cmake#L271

Reply:
This issue is being resolved here https://github.com/LLNL/blt/pull/494. However, this PR can still be merged without that BLT change.